### PR TITLE
Fixed traits and removed powergamey shit now you headmins better fucking re-enable it in the config or I'm gonna shoot up a school

### PR DESCRIPTION
**Pull request by Jujumatic**
Just merge it reeee 

but this will require a headmin to change the config, please do it

**Adds:**

As good trait:
- Neet
Along with our hippie traits

As bad trait:
- Blood Deficiency
- Blind
- Brain Tumor
- Deaf
- Heavy Sleeper
- Near Sighted
- Pacifist
- Paraplegic
- Poor Aim
- Prosopagnosia (The one where you can't see faces)
- Prosthetic Limb
- Reality Dissociation Syndrome
- Physically Obstructive
- Junkie
- Smoker
Along with our hippie traits

As neutral trait:
- Alcohol Tolerance
- Light Drinker
- Musician
- Photographer
- Spiritual
- Voracious (Can eat however much)
- Ageusia (Can't taste anything)
- Vegetarian
- Ananas Affinity (Like pineapples)
- Ananas Aversion (Dislike pineapples)
- Deviant Tastes (Like food most people do not)
- Monochromacy
- Family Heirloom
Along with our hippie traits

## Changelog
:cl:
balance: Removed powergamey stuff from TG traits
/:cl:

### CODE REVIEW

**Comment by yoyobatty** [comment]
Make a list of what it adds so it's easy for everyone to understand please.

File: hippiestation/code/datums/traits/neutral.dm [hunk 1]
```diff
@@ -1,84 +1,305 @@
...
-	if (locate(/datum/client_colour/monochrome) in quirk_holder.client_colours)
-		to_chat(quirk_holder, "<span class='warning'>You feel your vibrant vision having no effect because of your monochromacy!</span>")
-		sleep(20) //Don't want the two messages to pop up instantly, muh immershun
-		remove()
+	var/list/names = splittext(quirk_holder.real_name, " ")
+	var/family_name = names[names.len]
+
+	heirloom.AddComponent(/datum/component/heirloom, quirk_holder.mind, family_name)
+
+/datum/quirk/family_heirloom/on_process()
```
Comment (Carbonhell) [inline hippiestation/code/datums/traits/neutral.dm]: This quirk is pointless, we don't have moods do we?

**Review by Carbonhell — DISMISSED** [review]

**Comment by Carbonhell** [comment]
+1 to everything except heirloom, seems pointless

**Comment by Jujumatic** [comment]
Changed up family heirlooms, here is what it does now

If you have it on you, it occasionally sends you a message saying that you feel safe as you have the heirloom.

If you lose it, you get sent a message after a while saying that you realise you left your heirloom somewhere and that you should go get it. If you don't, after a while you start to get nervous and jittery without the heirloom. If you stay for much longer without the heirloom, you get more and more jittery and nervous, start nervously coughing. If you STILL don't have the heirloom at this point, you start getting extremely nervous and panicky and start coughing, screaming, falling over and throwing up. And if you still don't get the heirloom at this point, you get 30 seconds before you either have a heart attack or throw up two random organs.

So it's a negative trait, which gives you 1 point to spend on positive traits

Also it usually takes like 5 minutes before you get into the super panicky stage and your heirloom is super important. So you should never really lose it anyway

**Comment by Jujumatic** [comment]
Must admit it was pretty fun to code actually

**Comment by Carbonhell** [comment]
lol i like it, not sure how people who steal the heirloom should be treated though

**Comment by Jujumatic** [comment]
in this case i would say it'd be treated the same as if somebody stole your id

**Comment by earthmoonjune** [comment]
+1

traits are still for fags tho

**Comment by Carbonhell** [comment]
> in this case i would say it'd be treated the same as if somebody stole your id

but it can get you killed

**Comment by Jujumatic** [comment]
> > in this case i would say it'd be treated the same as if somebody stole your id
> 
> but it can get you killed

pro tip don't lose your heirloom

also i'm pretty sure people will soon realise no heirloom = death
so unless they want a catban they'd give you the heirloom back

**Comment by Carbonhell** [comment]
So does stealing and keeping the heirloom count as a kill?

**Comment by Jujumatic** [comment]
it should count as a kill, as the person will definitely die at some point

**Comment by DaydreamPuppeh** [comment]
+1

**Comment by TheMeatGod** [comment]
What exactly is the appeal of traits?

**Comment by Jujumatic** [comment]
@TheMeatGod it's just an extra gimmicky thing for people to have, it gives more random things to do and let's you make random/interesting scenarios. Great for people like me who love the gimmick stuff

**Review by ghost — APPROVED** [review]
Carbon hell's thing resolved

File: hippiestation/code/datums/traits/neutral.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/datums/traits/neutral.dm]: Juju did a new thing for this, so resolving

**Review by yoyobatty — APPROVED** [review]

**Pull request merged**